SwifterSwift.podspec

```diff
@@ -35,12 +35,14 @@ Pod::Spec.new do |s|
 
   # UIKit Extensions
   s.subspec 'UIKit' do |sp|
-    sp.source_files  = 'Sources/Extensions/UIKit/*.swift', 'Sources/Extensions/Shared/ColorExtensions.swift'
+    sp.dependency 'SwifterSwift/Shared/Color'
```

Comment:
Should/Could this (and the other instances) be
```
sp.dependency `Shared`
```
?
That way if we want to add more shared dependencies they would only need to be updated in one place.

Reply:
I am afraid that it can't be supported by the current version of Cocoapods. [Here](https://guides.cocoapods.org/syntax/podspec.html#subspec) is the usage of `supspec` with 'dependency'. And I revoke the `subspec 'Shared/Color'` and use its parent  `subspec 'Shared'` instead.